src/jupyterlabcontroller/storage/k8s.py

```diff
@@ -359,6 +359,8 @@ async def create_secret(
         immutable: bool = True,
     ) -> None:
         secret = V1Secret(
+            api_version="v1",
```

Comment:
kind and version are implicitly set, so I don't get why you're making them explicit, but OK I guess....

Reply:
Yeah, this will get fixed later. I was setting them solely for the test suite so that I have them available when testing the objects, but this work belongs in the mock layer rather than in the code. I'll move it in an upcoming PR. Sorry about that!